std/algorithm/searching.d

```diff
@@ -3462,6 +3462,59 @@ unittest
 }
 
 /**
+Computes the index of the first occurrence of `range`'s minimum (respectively maximum).
+
+Params:
+    pred = The ordering predicate to use to determine the extremum (minimum or
+           maximum) element.
+    range = The input range to search.
+
+Returns:
+    The index of the first encounter of the minimum (respectively maximum) in `range`. If the
+    range is empty, -1 is returned.
+ */
+ptrdiff_t minPosIndex(alias pred = "a < b", Range)(Range range)
```

Comment:
My bad. It seems that it does pass the unit test and it returns -1 even if return type is size_t. I assume it implicitly converts -1 to size_t, right?